src/testdata/postgres/compose.yaml.warnings

```diff
@@ -2,6 +2,8 @@
  ! service "no-ext": stateful service will lose data on restart; use a managed service instead
  ! service "no-ports": missing memory reservation; using provider-specific defaults. Specify deploy.resources.reservations.memory to avoid out-of-memory errors
  ! service "no-ports-override": missing memory reservation; using provider-specific defaults. Specify deploy.resources.reservations.memory to avoid out-of-memory errors
+ ! service "short-ports": ingress port without healthcheck defaults to GET / HTTP/1.1
```

Comment:
this warning is shown because the fixup happens too late

Reply:
Afaict this test doesn't consider fixup